arangod/VocBase/Validators.cpp

```diff
@@ -127,6 +129,7 @@ void ValidatorBase::toVelocyPack(VPackBuilder& b) const {
   VPackObjectBuilder guard(&b);
   b.add(StaticStrings::ValidationParameterMessage, VPackValue(_message));
   b.add(StaticStrings::ValidationParameterLevel, VPackValue(to_string(this->_level)));
+  b.add(StaticStrings::ValidationParameterType, VPackValue(this->type()));
```

Comment:
I actually do not need this anymore.
However I think knowing the Type of a schema could be benefitial in the future.
So i am happy with keeping and removing this line.